### PR TITLE
HDDS-5107. Add trace for containerReport

### PR DESCRIPTION
**Pull request by JacksonYao287**
## What changes were proposed in this pull request?

Add trace for containerReport to help debugging SCM OOM

## What is the link to the Apache JIRA

HDDS-5107

## How was this patch tested?

no need to test

### CODE REVIEW

File: hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/ContainerReportHandler.java [hunk 1]
```diff
@@ -118,6 +121,9 @@ public void onMessage(final ContainerReportFromDatanode reportFromDatanode,
     try {
       final List<ContainerReplicaProto> replicas =
           containerReport.getReportsList();
+      LOG.trace("begin processing ContainerReport of {} at {}," +
```
Comment (GlenGeng) [inline hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/ContainerReportHandler.java:124]: Please use `LOG.info` instead.

File: hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/ContainerReportHandler.java [hunk 2]
```diff
@@ -144,7 +150,9 @@ public void onMessage(final ContainerReportFromDatanode reportFromDatanode,
       LOG.error("Received container report from unknown datanode {}.",
           datanodeDetails, ex);
     }
-
+    LOG.trace("complete processing ContainerReport of {} at {}" +
+        "and processedContainerReportCount is {}", datanodeDetails.getUuid(),
+        Time.now(), processedContaienReportCount.addAndGet(1L));
```
Comment (GlenGeng) [inline hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/ContainerReportHandler.java:155]: Please calculate the latency and add to the `LOF.info` as well.

**Review by GlenGeng — COMMENTED** [review]
Thanks for the contribute. Some inline commens.

File: hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/ContainerReportHandler.java (hunk 1 above)
Comment (JacksonYao287) [reply hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/ContainerReportHandler.java]: ok, and a process latency is added too

**Pull request closed without merging**